samples/file-encryptor/enclave/Makefile

```diff
@@ -14,7 +15,9 @@ all:
 
 build:
 	@ echo "Compilers used: $(CC), $(CXX)"
-	oeedger8r ../fileencryptor.edl --trusted
+	oeedger8r ../fileencryptor.edl --trusted \
+		--search-path $(INCDIR) \
+		--search-path $(INCDIR)/openenclave/edl/sgx
```

Comment:
put SGX under some SGX-specific condition

Reply:
Closing this and others based on discussion with @HernanGatta